CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+This Code of Conduct applies to Code.gov, its repositories, communications, events and anyone participating in the Code.gov community [link to about page]. Your use of Code.gov means you accept and agree to abide by this Code of Conduct, which we may revise periodically. You should review this page from time to time. 
+
+## What we expect
+
+We expect all visitors and contributors to Code.gov to have a safe and engaging experience that is beneficial to our community. Members of our community have a diverse set of experiences and skillsets. As you engage our community, please ensure you are:  
+
+- __Considerate__: as an open source community, what you do will affect others. Ensure what you do is in the best interest of the project, and community at large. 
+- __Collaborative__: the essences of open source is collaborative in nature. Attempt to honor the spirit of open source by working together before conflict.
+- __Patient__: Remember that open source is a culture change in government and while Code.gov is a testament to that change, change is occurring thanks to many in this community. Be patient if your question or issue is not answered right away. Many are juggling multiple duties. Contributing to open-source projects is often an after hours affair. 
+- __Be Mindful__: Be aware of your actions, others actions. Treat others with dignity and respect --the way you want to be treated. Be kind. 
```

Comment:
Grammatical issues here, too:
__Be Mindful__: Be aware of your actions.  Treat others with dignity and respect -- the way you want to be treated. Be kind.

Reply:
👍